phase1/pallavi/file_processing.py

```diff
@@ -0,0 +1,44 @@
+import os
+from pathlib import Path
+import pandas as pd
+from sqlalchemy import Column, Integer, String, Float, BigInteger, DateTime, TEXT
+from sqlalchemy.exc import SQLAlchemyError
+import logging
+
+def list_files(directory_path):
+    return [file for file in directory_path.iterdir() if file.is_file()]
+
+def check_directory_structure(directory_path):
```

Comment:
what is the use of this?

Reply:
To check which types of files are present eg: data1.csv  ,  data2.json etc